### PR TITLE
Add Remix wizard instructions

### PR DESCRIPTION
**Pull request by onurtemizkan**
Fixes: #7727
Continues: https://github.com/getsentry/sentry-docs/pull/7728

Can't push commits to the original PR, so opened a new one.

### CODE REVIEW

File: src/platform-includes/getting-started-install/javascript.remix.mdx [hunk 1]
```diff
@@ -1,7 +1,24 @@
...
+- create two files in the root directory of your project, `entry.client.tsx` and `entry.server.tsx` (if they don't exist yet)
+- add the default `Sentry.init()` for the client in `entry.client.tsx` and the server in `entry.server.tsx`
+- create `.sentryclirc` with an auth token to upload source maps (this file is automatically added to `.gitignore`)
+
+If you use [Remix future flags](https://remix.run/docs/en/main/pages/api-development-strategy#current-future-flags), the wizard will instrument your application accordingly to support Remix v2 features.
+
+After the wizard setup is completed, the SDK will automatically capture unhandled errors, and monitor performance.
+You can also <PlatformLink to="/usage/">manually capture errors</PlatformLink>.
+
+To learn how to connect your app to Sentry and deploy it on Vercel, see the [Vercel integration](/product/integrations/deployment/vercel/).
```
Comment (AbhiPrasad) [inline src/platform-includes/getting-started-install/javascript.remix.mdx:18]: We can remove this for Remix users.

```suggestion
```

File: src/platforms/javascript/guides/remix/manual-setup.mdx [hunk 2]
```diff
@@ -0,0 +1,209 @@
+---
+title: Manual Setup
+sidebar_order: 1
+description: "Learn how to set up the SDK manually."
+---
+
+If you can't (or prefer not to) run the [automatic setup](/platforms/javascript/guides/remix/#install), you can follow the instructions below to configure your application.
+
+## Installation
```
Comment (lizokm) [inline src/platforms/javascript/guides/remix/manual-setup.mdx:9]:
```suggestion
## Installation

Get started by installing the below:
```

File: src/platforms/javascript/guides/remix/manual-setup.mdx [hunk 3]
```diff
@@ -0,0 +1,209 @@
...
+
+```bash {tabTitle:pnpm}
+pnpm add @sentry/remix
+```
+
+## Configuration
+
+To use this SDK, initialize Sentry in your Remix entry points for both the client and server.
+
+Create two files in the root directory of your project, `entry.client.tsx` and `entry.server.tsx` (if they don't exist yet). In these files, add your initialization code for the client-side SDK and server-side SDK, respectively.
```
Comment (lizokm) [inline src/platforms/javascript/guides/remix/manual-setup.mdx:27]:
```suggestion
Create two files in the root directory of your project, `entry.client.tsx` and `entry.server.tsx` (if they don't already exist). Add your initialization code in these files for the client-side and server-side SDK, respectively.
```

File: src/platforms/javascript/guides/remix/manual-setup.mdx [hunk 4]
```diff
@@ -0,0 +1,209 @@
...
+withSentry(App, {
+  errorBoundaryOptions: {
+    fallback: <p>An error has occurred</p>,
+  },
+});
+```
+
+## Custom Express Server
+
+If you use a custom Express server in your Remix application, you should wrap your [`createRequestHandler` function](https://remix.run/docs/en/v1/other-api/adapter#createrequesthandler) manually with `wrapExpressCreateRequestHandler`. This is not required if you use the built-in Remix App Server.
```
Comment (lizokm) [inline src/platforms/javascript/guides/remix/manual-setup.mdx:142]:
```suggestion
If you use a custom Express server in your Remix application, you should wrap your [`createRequestHandler` function](https://remix.run/docs/en/v1/other-api/adapter#createrequesthandler) manually with `wrapExpressCreateRequestHandler`. This isn't necessary if you're using the built-in Remix App Server.
```

File: src/platforms/javascript/guides/remix/manual-setup.mdx [hunk 5]
```diff
@@ -0,0 +1,209 @@
...
+// ...
+
+const createSentryRequestHandler =
+  wrapExpressCreateRequestHandler(createRequestHandler);
+
+// Use createSentryRequestHandler like you would with createRequestHandler
+app.all("*", createSentryRequestHandler(/* ... */));
+```
+
+## Remix v2 features
```
Comment (lizokm) [inline src/platforms/javascript/guides/remix/manual-setup.mdx:163]:
```suggestion
## Remix v2 Features
```

File: src/platforms/javascript/guides/remix/manual-setup.mdx [hunk 6]
```diff
@@ -0,0 +1,209 @@
...
+
+_Available from SDK version 7.59.0_
+
+[Remix v2](https://remix.run/docs/en/main/pages/v2) will introduce new features that require additional configuration to work with Sentry. These features are also available from version [1.17.0](https://github.com/remix-run/remix/releases/tag/remix%401.17.0) with [future flags](https://remix.run/docs/en/main/pages/api-development-strategy#current-future-flags).
+
+### v2 ErrorBoundary
+
+To capture errors from [v2 ErrorBoundary](https://remix.run/docs/en/main/route/error-boundary-v2), you should define your own `ErrorBoundary` in `root.tsx` and use `Sentry.captureRemixErrorBoundaryError` inside of it. You can also create route-specific error capturing behaviour by defining `ErrorBoundary` in your route components. The `ErrorBoundary` you define in `root.tsx` will be used as a fallback for all routes.
```
Comment (lizokm) [inline src/platforms/javascript/guides/remix/manual-setup.mdx:171]:
```suggestion
To capture errors from [v2 ErrorBoundary](https://remix.run/docs/en/main/route/error-boundary-v2), you should define your own `ErrorBoundary` in `root.tsx` and use `Sentry.captureRemixErrorBoundaryError` inside of it. You can also create route-specific error capturing behavior by defining `ErrorBoundary` in your route components. The `ErrorBoundary` you define in `root.tsx` will be used as a fallback for all routes.
```

File: src/platforms/javascript/guides/remix/manual-setup.mdx [hunk 7]
```diff
@@ -0,0 +1,209 @@
...
+
+  captureRemixErrorBoundaryError(error);
+
+  return <div> ... </div>;
+};
+```
+
+## v2 Server-side Errors
+
+When using `v2_errorBoundary` future flag, Sentry can't capture your server-side errors automatically. Instead, define a [`handleError`](https://remix.run/docs/en/main/file-conventions/entry.server#handleerror) function in your server entry point. Then, you should use `Sentry.captureRemixServerError` to capture errors in your server-side code.
```
Comment (lizokm) [inline src/platforms/javascript/guides/remix/manual-setup.mdx:187]:
```suggestion
Sentry won't be able to capture your server-side errors automatically if you're using the`v2_errorBoundary` future flag. To work around this, define a [`handleError`](https://remix.run/docs/en/main/file-conventions/entry.server#handleerror) function in your server entry point. Then use `Sentry.captureRemixServerError` to capture errors in your server-side code.
```

File: src/platforms/javascript/guides/remix/manual-setup.mdx [hunk 8]
```diff
@@ -0,0 +1,209 @@
...
+  if (error instanceof Error) {
+    Sentry.captureRemixServerException(error, "remix.server", request);
+  } else {
+    // Optionally capture non-Error objects
+    Sentry.captureException(error);
+  }
+}
+```
+
+Once you've done this set up, the SDK will automatically capture unhandled errors and promise rejections, and monitor performance in the client. You can also [manually capture errors](/platforms/javascript/guides/remix/usage).
```
Comment (lizokm) [inline src/platforms/javascript/guides/remix/manual-setup.mdx:203]:
```suggestion
After you've completed this setup, the SDK will automatically capture unhandled errors and promise rejections, and monitor performance in the client. You can also [manually capture errors](/platforms/javascript/guides/remix/usage).
```

File: src/platforms/javascript/guides/remix/manual-setup.mdx [hunk 9]
```diff
@@ -0,0 +1,209 @@
...
+
+<Note>
+
+You can refer to [Remix Docs](https://remix.run/docs/en/v1/guides/envvars#browser-environment-variables) to learn how to use your Sentry DSN from environment variables.
```
Comment (lizokm) [inline src/platforms/javascript/guides/remix/manual-setup.mdx:207]:
```suggestion
You can refer to [Remix Docs](https://remix.run/docs/en/v1/guides/envvars#browser-environment-variables) to learn how to use your Sentry DSN with environment variables.
```

**Review by lizokm — APPROVED** [review]
Made some suggestions, but otherwise this looks great!

File: src/platforms/javascript/guides/remix/manual-setup.mdx [hunk 10]
```diff
@@ -0,0 +1,209 @@
...
+// or
+
+withSentry(App, {
+  errorBoundaryOptions: {
+    fallback: <p>An error has occurred</p>,
+  },
+});
+```
+
+## Custom Express Server
```
Comment (Lms24) [inline src/platforms/javascript/guides/remix/manual-setup.mdx:140]: l: I'd prefer this section and the v2 support section to switch places unless there's a concrete reason to have custom express server instructions come first(?) My thinking is that showing that we support v2 (future flags) is more important than custom servers.

File: src/platforms/javascript/guides/remix/manual-setup.mdx (hunk 2 above)
Comment (Lms24) [reply src/platforms/javascript/guides/remix/manual-setup.mdx:9]: l: Would just suggest a slight change here:
```suggestion
## Installation

Get started by installing the Sentry Remix SDK:
```

**Review by Lms24 — APPROVED** [review]
LGTM once the suggestions are applied, thanks Onur!

**Review by Lms24 — CHANGES_REQUESTED** [review]
Only requesting changes so we don't merge this until https://github.com/getsentry/sentry-wizard/issues/433 is resolved. Let's only publish this once the wizard is fully ready.

File: src/platform-includes/getting-started-install/javascript.remix.mdx [hunk 11]
```diff
@@ -1,7 +1,22 @@
...
-yarn add @sentry/remix
+```bash
+npx @sentry/wizard@latest -i remix
 ```
+
+The wizard will prompt you to log in to Sentry. It will then automatically do the following steps for you:
+
+- create two files in the root directory of your project, `entry.client.tsx` and `entry.server.tsx` (if they don't already exist).
+- add the default `Sentry.init()` for the client in `entry.client.tsx` and the server in `entry.server.tsx`.
+- create `.sentryclirc` with an auth token to upload source maps (this file is automatically added to `.gitignore`).
```
Comment (Lms24) [inline src/platform-includes/getting-started-install/javascript.remix.mdx:11]:
```suggestion
- create `.sentryclirc` with an auth token to upload source maps (this file is automatically added to `.gitignore`).
- adjust your `build` script in `package.json` to automatically upload source maps to Sentry.
```

**Review by Lms24 — APPROVED** [review]
We released the wizard with the org/project fix yesterday so this is ready to go. Just one small suggestion but otherwise ready to 🚢

File: src/platforms/javascript/guides/remix/manual-setup.mdx [hunk 12]
```diff
@@ -0,0 +1,211 @@
...
+  dsn: "___PUBLIC_DSN___",
+
+  // Set tracesSampleRate to 1.0 to capture 100%
+  // of transactions for performance monitoring.
+  // We recommend adjusting this value in production
+  tracesSampleRate: 1.0,
+});
+```
+
+Initialize Sentry in your entry point for the server to capture exceptions and get performance metrics for your [`action`](https://remix.run/docs/en/v1/api/conventions#action) and [`loader`](https://remix.run/docs/en/v1/api/conventions#loader) functions. You can also initialize Sentry's database integrations, such as <Link to="/platforms/node/performance/database/opt-in/#prisma-orm-integration">Prisma</Link>, to get spans for your database calls.
```
Comment (shanamatthews) [inline src/platforms/javascript/guides/remix/manual-setup.mdx:84]: Looks like these links have moved

```suggestion
Initialize Sentry in your entry point for the server to capture exceptions and get performance metrics for your [`action`](https://remix.run/docs/en/1.19.3/route/action) and [`loader`](https://remix.run/docs/en/1.19.3/route/loader) functions. You can also initialize Sentry's database integrations, such as <Link to="/platforms/node/performance/database/opt-in/#prisma-orm-integration">Prisma</Link>, to get spans for your database calls.
```

**Review by shanamatthews — APPROVED** [review]

File: src/platforms/javascript/guides/remix/manual-setup.mdx (hunk 12 above)
Comment (onurtemizkan) [inline src/platforms/javascript/guides/remix/manual-setup.mdx:84]: @shanamatthews - v2.0.0 is published so we can use `main` here instead. 

```suggestion
Initialize Sentry in your entry point for the server to capture exceptions and get performance metrics for your [`action`](https://remix.run/docs/en/main/route/action) and [`loader`](https://remix.run/docs/en/main/route/loader) functions. You can also initialize Sentry's database integrations, such as <Link to="/platforms/node/performance/database/opt-in/#prisma-orm-integration">Prisma</Link>, to get spans for your database calls.
```

**Pull request merged**